compiler/damlc/tests/src/package-pattern-match-perf.sh

```diff
@@ -28,4 +28,4 @@ $DAMLC build --project-root $DIR/dep
 # does not come with larger memomry or stack usage.  However, the slow
 # down is so big that this times out the 300s Bazel timeout whereas
 # the fixed version runs in about 30s.
-$DAMLC build --project-root $DIR/main +RTS -s -M100M -K1M -N1
+$DAMLC build --project-root $DIR/main +RTS -s -M150M -K1M -N1
```

Comment:
For other reviewers: We’ve discussed this and this should be fine. The bound was pretty tight and 150M isn’t that unreasonable for this test.